build/template-OneBranch-CI-libsandsamples.yaml

```diff
@@ -71,10 +71,6 @@ steps:
   inputs:
     script: 'nuget locals all -clear'
 
-- ${{ if eq(parameters.Mode, 'build') }}:
```

Comment:
Are we no longer targeting these Android SDKs in our build?

Reply:
we are targeting MAUI and we do have a MAUI android install step.